### DISCUSSION TITLE
Glance on Pinned Tabs

### DISCUSSION THREAD

**Discussion by Anthonyy232** [zen-browser/desktop#2431] [Ideas]
Hi, it would be awesome to allow automatically doing glance on a link if it is from a pinned tab similar to Arc.

Of course, this will be an option under the current selectors.
![image](https://github.com/user-attachments/assets/973e5c25-c7de-4915-ac7f-d3d98831d724)

**Comment by mr-cheffy** [comment]
Yes, this will be available only with favourite tabs, thanks!

**Comment by Milkman337** [comment]
I think there should be an option where you can enable it for None, Favourite tabs or all tabs.

**Comment by FBEPcode** [comment]
+1 for @Milkman337's  suggestion. I understand that Glance is useful to a lot of folks. I prefer not to use it, but I like Essentials tabs. I would love to see the option to turn Glance off on those as well.
